bigbluebutton/bigbluebutton
# Fix for safari and ios specific code

- Fixes Presentation on Safari
- Fixes Dropdowns on iOS
- Implements handlers for the iOS app
  - Handle join voice conference on iOS app
  - Handle join Breakout on iOS app

## To test on the iOS app
Download Testflight on your iOS device, and select `redeem`. You should have received an email invitation to Apple's Testflight which contains instructions and the code you should redeem to install the app on the device. Once you insert the code and proceed with the instructions you will be prompted to install the app on the device.
